lovelace-eink-theme.yml

```diff
@@ -36,13 +32,11 @@ eink:
 
   ### Cards ###
   paper-card-background-color: "#ffffff"
-  paper-listbox-background-color: "#ffffff"
+  # paper-listbox-background-color has been removed in HA 2025.5+
```

Comment:
Remove comments like this. We don't care when it was removed. Check other comments as well.